### PR TITLE
[8579] Finis elkjop doc

### PR DESCRIPTION
**Pull request by DavidVergnaultMoank**

### CODE REVIEW

File: specifications.yml [hunk 1]
```diff
@@ -41,16 +41,16 @@ paths:
...
         - oAuth2ClientCredentials: []
 
-  /application/customer/{customerApplicationId}:
+  /application/customer/{applicationId}:
     get:
       tags:
       - Application endpoints
-      summary: Get a customer application
-      description: Fetch a customer application
+      summary: Get a customer
```
Comment (BlackEydPanda) [inline specifications.yml:48]: Wait why change this to simply get a customer, arent we fetching an application?

**Review by BlackEydPanda — DISMISSED** [review]

File: specifications.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ info:
     The Fimento Open API can be used by Fimento's partners in order to provide financial products to their end customers within their own application/website.
 
     # Test objects
-    Test objects can be used in the staging environment in order to test specific flows. If one of the organsiation of the application has its `organisationNumber` equal to one of the test object, the `applicationStatus` returned by Fimento for the create application endpoint will always be the same one as described in the following table:
+    Test objects can be used in the staging environment in order to test specific flows. If one of the organisations of the application has its `organisationNumber` equal to one of the test objects, the `applicationStatus` returned by Fimento for the create application endpoint will always be the same one as described in the following table:
```
Comment (Erik-Nilsson-Fimento) [inline specifications.yml]: This should be singular, there is always only one organisation per application

Comment (Erik-Nilsson-Fimento) [reply specifications.yml]: So maybe `If the organisation of the customer of the application....`

File: specifications.yml (hunk 1 above)
Comment (Erik-Nilsson-Fimento) [reply specifications.yml:48]: From the point of view of the user calling the endpoint there is no difference 🙂 Elkjop will call this endpoint expecting the data of the customer they just created, calling it application might be a bit confusing in that sense

Comment (Erik-Nilsson-Fimento) [reply specifications.yml:48]: Even though, yes you are correct in that it is an application in practice 😄

**Review by Erik-Nilsson-Fimento — DISMISSED** [review]

**Review by BlackEydPanda — APPROVED** [review]

**Pull request merged**